.gitattributes

```diff
@@ -1,7 +1,7 @@
+/.github/           export-ignore
```

Comment:
We probably do not need to export any of the files contained in this directory either.